packages/nocodb/src/lib/constants/index.ts

```diff
@@ -1,2 +1,5 @@
 export const NC_LICENSE_KEY = 'nc-license-key';
 export const NC_APP_SETTINGS = 'nc-app-settings';
+export const NC_FILE_FIELD_SIZE = process.env['NC_ATTACHMENT_FIELD_SIZE']
+  ? Number(process.env['NC_ATTACHMENT_FIELD_SIZE'])
+  : 20 * 1024 * 1024; // 20 MB
```

Comment:
If `process.env['NC_ATTACHMENT_FIELD_SIZE']` is not a number, says `abc`, then  `NC_ATTACHMENT_FIELD_SIZE` would be `NaN`.

You can change this to the following so that any invalid value would turn to the default one.

```
+process.env['NC_ATTACHMENT_FIELD_SIZE'] || 20 * 1024 * 1024
```